csrc/executor.cpp

```diff
@@ -1384,6 +1374,19 @@ std::vector<FusionExecutor::GlobalBufferInfo> FusionExecutor::
   return outputs;
 }
 
+std::vector<at::Tensor> allocOutputSpace(
+    const at::ArrayRef<c10::IValue>& inputs,
+    kir::Kernel* kernel,
```

Comment:
Question, not request for change: I guess `Fusion` should be good enough here (of course, refactors required). allocOutputSpace doesn't use anything that's only in the lowered Kernel but not in the Fusion. Correct? 

cc @Priya2698

Reply:
Let's change the parameter to `Fusion*` (assuming the change is trivial). Having `Kernel` indicates it needs lowering.

Reply:
That makes sense! I didn't do it upfront bc I was lazy, but it is done now! :)